frontend/src/pages/ArtItemPage/CreateArtItemPage.js

```diff
@@ -1,11 +1,135 @@
-import React from "react";
+import React, { useReducer } from "react";
+import { useAuth } from "../../auth/useAuth";
 
-function CreateArtItemPage() {
+import { TextField, Typography, Stack} from "@mui/material";
+import GenericCardLayout from "../../layouts/GenericCardLayout";
+import ImageUploader from '../../components/ImageUploader';
+import LoadingButton from '../../components/LoadingButton';
+
+function CreateArtItemForm(props) {
+
+    const [formInput, setFormInput] = useReducer(
+      (state, newState) => ({ ...state, ...newState }),
+      {
+        name: "",
+        description: "",
+        category: "",
+        labels: ""
+      }
+    );
+    const [selectedImage, setSelectedImage] = React.useState(null);
+    const [isLoading, setIsLoading] = React.useState(false);
+  
+    const formatFormInput = (imageId) => {
+      return {
+        artItemInfo: {
+          name: formInput.name,
+          description: formInput.description,
+          category: formInput.category,
+          labels: formInput.labels,
+          imageId: imageId
+        },
+        lastPrice: 0
+      }
+    }
+  
+    const { postRequestWithImage } = useAuth()
+  
+    const handleSubmit = event => {
+      event.preventDefault();
+      setIsLoading(true)
+      
+      postRequestWithImage(
+        "/api/art_item",
+        selectedImage,
+        formatFormInput,
+        (data) => "/artitem/"+data.id
+      )
+    };
+    
+    const handleInput = event => {
+      const name = event.target.name;
+      const newValue = event.target.value;
+      setFormInput({ [name]: newValue });
+    };
+  
     return (
-        <div>
-            This page is protected
-        </div>
-    )
+      <>
+        <Typography variant="h5" component="h3">
+          New Art Item
+        </Typography>
+  
+        <form onSubmit={handleSubmit}>
+          <Stack>
+            <TextField
+              required
+              id="outlined-required"
+              label="Art Item Name"
+              name="name"
+              defaultValue={formInput.name}
+              onChange={handleInput}
+              sx = {{marginY: 1}}
+            />
+            <TextField
+              required
+              id="outlined-required"
```

Comment:
We can add `multiline` prop for better user experience as descriptions can be long